messenger.rst

```diff
@@ -844,6 +844,12 @@ options.
 AMQP Transport
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 5.2
+
+    Starting from Symfony 5.2, the AMQP transport can handle AMQPS DSN (e.g.  ``amqps://``).
+    Be aware that using it without using CA certificate can throw an exception.
```

Comment:
This sentence isn't really helpful. How do I set this CA certificate? We should either not mention it (and rely on Symfony to throw an exception that will educate the user on what to do next) or fully show how to use amps.

Reply:
I've added an entry about it, wdyt ?